build/optaplanner-build-parent/pom.xml

```diff
@@ -147,12 +150,33 @@
         <artifactId>h2</artifactId>
         <version>2.1.210</version>
       </dependency>
+      <!-- optaplanner-operator -->
+      <dependency>
+        <groupId>io.quarkiverse.operatorsdk</groupId>
+        <artifactId>quarkus-operator-sdk</artifactId>
+        <version>${version.io.quarkiverse.operatorsdk}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.javaoperatorsdk</groupId>
+        <artifactId>operator-framework-core</artifactId>
+        <version>${version.io.javaoperatorsdk}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.strimzi</groupId>
+        <artifactId>api</artifactId>
```

Comment:
(not an issue) Why doesn't quarkus-bom define a io.strimzi:api version? That's weird.

Reply:
Strimzi is not anyhow related to Quarkus; I would be surprised if the quarkus-bom did define its version. This artifact is interesting only to people who need to programmatically approach the Strimzi operator from Java.